src/KronPhotometry.cc

```diff
@@ -432,6 +436,15 @@ KronFluxAlgorithm::KronFluxAlgorithm(KronFluxControl const & ctrl, std::string c
         {"flag_bad_shape", "shape for measuring Kron radius is bad; used PSF shape"},
     }};
     _flagHandler = meas::base::FlagHandler::addFields(schema, name, flagDefs.begin(), flagDefs.end());
+    metadata.add(name + "_fixed", ctrl.fixed);
+    metadata.add(name + "_nSigmaForRadius", ctrl.nSigmaForRadius);
+    metadata.add(name + "_nIterForRadius", ctrl.nIterForRadius);
+    metadata.add(name + "_nRadiusForFlux", ctrl.nRadiusForFlux);
+    metadata.add(name + "_maxSincRadius", ctrl.maxSincRadius);
+    metadata.add(name + "_minimumRadius", ctrl.minimumRadius);
+    metadata.add(name + "_enforceMinimumRadius", ctrl.enforceMinimumRadius);
+    metadata.add(name + "_useFootprintRadius", ctrl.useFootprintRadius);
+    metadata.add(name + "_smoothingSigma", ctrl.smoothingSigma);
```

Comment:
I agree with all of you...  No configuration, except nRadiusForFlux is something you need to understand the measurements (e.g. plotting the aperture used).